spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -83,11 +84,13 @@ class ReverseIPTagMulticastSourceMachineVertex(
     """ A model which allows events to be injected into SpiNNaker and\
         converted in to multicast packets.
 
+    :param str label: The label of this vertex
     :param ~pacman.model.graphs.common.Slice vertex_slice:
         The slice served via this multicast source
```

Comment:
The  vertex_slice: and app_vertex params need to be documented as or none to do this do.
    :param vertex_slice:
        The slice served via this multicast source
    :type vertex_slice: ~pacman.model.graphs.common.Slice or None

As or can not be used in the :param documentation